core/src/main/web/app/helpers/helper.js

```diff
@@ -63,7 +63,112 @@
       newSession: function(empty) {
         return bkCoreManager.newSession(empty);
       },
-
```

Comment:
Yep! Its in one of the most recent commits. bkElectron lives in utils/electron.js